INSTALL.md

```diff
@@ -0,0 +1,105 @@
+# Installation
+
+## Table of contents
+
+- [Arch-based distributions](#arch-based-distributions)
+- [Debian](#debian)
+- [Fedora](#fedora)
+- [OpenMandriva](#openmandriva)
+- [openSUSE](#opensuse)
+- [Pop!_OS](#pop_os)
+- [Ubuntu-based distributions](#ubuntu-based-distributions)
+- [AppImage](#appimage)
+- [Flatpak](#flatpak)
+- [Prebuilt releases](#prebuilt-releases)
+- [Source](#source)
+	- [Building](#building)
+	- [Installing](#installing)
+
+## Arch-based distributions
+[`gamehub-git`](https://aur.archlinux.org/packages/gamehub-git/) and [`gamehub`](https://aur.archlinux.org/packages/gamehub/) are available in AUR.
+
+## Debian
+Unfortunately, there is no package available in the Debian repository, so it will be required to run the [`/scripts/build.sh build_deb`](../scripts/build.sh#L171-L210) script. The script will install the dependencies and GameHub.
```

Comment:
I haven't actually tried it on Debian, but I think packages for Ubuntu should install on Debian if dependencies are available in repos.

Reply:
that may be true, but PPAs have been prone to break in Debian since they were made specifically for Ubuntu. i wouldn't recommend adding PPAs if you are risking breakage

Reply:
It should be possible to just download prebuilt .deb from releases page and install it on Debian without adding any repos (if all of dependencies are actually available in Debian repos).

Reply:
i'll install a Debian VM and will test. hope it works

Reply:
*it's taking a while to install*

Reply:
sad. seems like Debian doesn't want to boot after waiting an hour and half of installing

Reply:
tried a livecd, it doesn't boot either. i can't really test

Reply:
> I haven't actually tried it on Debian, but I think packages for Ubuntu should install on Debian if dependencies are available in repos.

i would suggest we can add a warning that the .deb hasn't been tested in Debian, so users may experience issues

Reply:
I can test
Gimme a few mins

Reply:
Used the PPA, nothing broke, the commands are different though. I think that the granite package could cause some packages to break, not sure though. (btw I have no idea how to do the suggestions thing lol

## Debian
```
sudo apt install dirmngr
sudo sh -c "echo 'deb http://ppa.launchpad.net/tkashkin/gamehub/ubuntu focal main' > /etc/apt/sources.list.d/gamehub-ppa.list
sudo apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 5B63B42CE14BA47CC1B69E7C32B600D632AF380D
sudo apt install com.github.tkashkin.gamehub
``